examples/ErrorHandling/HandleKeywordPolicyViolations.php

```diff
@@ -237,6 +227,10 @@ private static function fetchExemptPolicyViolationKeys(GoogleAdsException $googl
                         $policyViolationDetailsKey->getViolatingTextUnwrapped(),
                         PHP_EOL
                     );
+                } else {
+                    print "No exemption request is sent because your keyword contained some"
+                        . " non-exemptible policy violations." . PHP_EOL;
+                    throw $googleAdsException;
                 }
```

Comment:
Is this message actually accurate? It's just for one error.
We still have other error in this for loop to check.
If there is at least one error that has  exemptible policy violation, `$exemptPolicyViolationKeys` is still added, and the policy exemption request will still be sent.

Reply:
Good catch!